modules/puzzles/module.py

```diff
@@ -33,7 +33,8 @@ def locations_with(self, symbols):
         """
         results = util.locations_containing_symbols(symbols)
         self.respond('Er zijn %i resultaten die de symbolen \'%s\' bevatten' % (len(results), symbols))
-        self.respond(', '.join([x.name for x in results]))
+        if(results):
+            self.respond(', '.join([x.name for x in results]))
```

Comment:
Misschien(en dat is een beetje een losse feature), is het een goed idee om bij geen/weinig resultaten ook te zoeken naar alle one offs? i.e. misschien hebben we een letter verkeerd opgelost, en maakt die een bruteforce oplossing onmogelijk.

Reply:
Zou dat niet teveel oplossingen geven? Met 10 sets coordinaten vind ik de bruikbaarheid al laag.

Reply:
Daarom ook bij geen/weinig resultaten.

Reply:
Volgens mij krijg je bij de one offs van geen resultaten alsnog te veel resultaten om bruikbaar te zijn, maar dat zou ik moeten onderzoeken. Lijkt me goed als extra feature voor later, net als het aangeven van onbekende sets getallen